SparkPost/2web2ui
# SA-1329: Filters Drawer for Report Builder

### What Changed
 - Filters changed to be a drawer
- Filters are selected based on type selection

### How To Test
 - With a hibana account, go to Summary Report
 - Click "Add Filters" button
- Drawer should open

### To Do
- [ ] Drawer should be preserved when switching to another tab
- [ ] Move the query paramater behavior out of the share button
- [ ] Confirm the clear behavior with Jon
